docs/src/pages/premium-themes/instapaper/theme/instapaper/components/tabs.js

```diff
@@ -15,52 +15,30 @@ export default ({ attach, theme, nest, ICON, TAB }) => ({
   },
   MuiTab: {
     root: {
-      lineHeight: 'inherit',
+      minHeight: 54,
+      fontWeight: 600,
       minWidth: 0,
-      marginRight: theme.spacing(1),
-      marginLeft: theme.spacing(1),
       [theme.breakpoints.up('md')]: {
         minWidth: 0,
-        marginRight: 30,
-        marginLeft: 30,
-      },
-      [attach(TAB.selected)]: {
-        [nest(TAB.label)]: {
-          fontWeight: 600,
-        },
-        '& *': {
-          color: '#262626 !important',
-        },
       },
     },
     labelIcon: {
-      minHeight: 53,
-      paddingTop: 0,
-      '& .MuiTab-wrapper': {
-        flexDirection: 'row',
-      },
-      [nest(ICON.root)]: {
+      minHeight: null,
+      paddingTop: null,
+      '& $wrapper :first-child': {
```

Comment:
Oh this rule applies to all first-childs in the tree. This messes things up, if you are using icons and labels. 

`'& $wrapper :first-child' === '& $wrapper *:first-child'`

I think you want to omit the space here.

Reply:
In my case I am using Tabs with Labels, Icons and Badges the above rule is applied to all of them.

Reply:
Are you using this custom theme? Yeah, I'm happy to try the change.

Reply:
Oh actually not. I added this comment to the wrong part of the code 😬. Nevertheless the issue stays the same. You might want look at this, too.

https://github.com/mui-org/material-ui/blob/8a13bc294e04c1af8fd5cf69a287132974b81577/packages/material-ui/src/Tab/Tab.js#L37-L39

Reply:
Here is an example: https://codesandbox.io/s/53o696672p

Reply:
What about? 
```js
'& $wrapper > * :first-child': {
```

Reply:
lgtm! 😍

Reply:
Awesome! Do you want to submit a pull request? :)

Reply:
Happy to!

Reply:
Done #14844